src/main/scala/nl/knaw/dans/bag/DansBag.scala

```diff
@@ -462,8 +462,7 @@ trait DansBag {
    * will move the file if both source and destination are on the same mount, but fail otherwise.
    *
    * When `src` is a directory, each file in that directory will be copied into the payload directory.
-   * If either `MOVE` or `ATOMIC_MOVE` is selected for `importOption`, importing a directory structure
-   * is not allowed.
+   * If `MOVE` is selected for `importOption`, importing a directory structure is not allowed.
```

Comment:
So it is allowed now to `ATOMIC_MOVE` a directory?

Reply:
Indeed.